srtcore/api.h

```diff
@@ -232,7 +232,7 @@ friend class CRendezvousQueue;
       /// @return If the new connection is successfully created: 1 success, 0 already exist, -1 error.
 
    int newConnection(const SRTSOCKET listen, const sockaddr_any& peer, const CPacket& hspkt,
-           CHandShake& w_hs, int& w_error);
+           CHandShake& w_hs, int& w_error, CUDT*& w_acpu);
```

Comment:
Please update the doxyxeg-style comments above the function: a description of `w_acpu` and `w_error` is missing.